generic_io/rl6/firmware.lua

```diff
@@ -2,7 +2,65 @@ local CHANNELS_NUMBER = 6
 
 local channel_arg_required_errmsg = 'channel arg is required and must be a number'
 
+function set_default_state()
+  local values, err = config.read_all()
+  if err then
+    enapter.log('cannot read config: ' .. tostring(err), 'error. Opening all channels.')
```

Comment:
Тут, видимо, должно быть так:
```suggestion
    enapter.log('cannot read config: ' .. tostring(err) .. '. Opening all channels.', 'error')
```